File: src/main/resources/output/IFC4X3_csharp/Entities/IfcCoordinateReferenceSystem/IfcProjectedCRS/IfcProjectedCRS.cs
public class IfcProjectedCRS : IfcCoordinateReferenceSystem
{
    public IfcIdentifier VerticalDatum { get; set; }
    public IfcIdentifier MapProjection { get; set; }
    public IfcIdentifier MapZone { get; set; }
    public IfcNamedUnit MapUnit { get; set; }

    // === WHERE CLAUSES ===
    // MapUnitIsLength : NOT(EXISTS(SELF.MapUnit)) OR (SELF.MapUnit.UnitType = IfcUnitEnum.LENGTHUNIT)
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcCoordinateReferenceSystem/IfcProjectedCRS/IfcProjectedCRS.py
class IfcProjectedCRS(IfcCoordinateReferenceSystem):
    def __init__(self):
        self.VerticalDatum: IfcIdentifier = None
        self.MapProjection: IfcIdentifier = None
        self.MapZone: IfcIdentifier = None
        self.MapUnit: IfcNamedUnit = None

    # === WHERE CLAUSES ===
    # MapUnitIsLength : NOT(EXISTS(SELF.MapUnit)) OR (SELF.MapUnit.UnitType = IfcUnitEnum.LENGTHUNIT)
